Data_Structures_Answers.md

```diff
@@ -1,18 +1,25 @@
 Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
+    O(n) time complexity
```

Comment:
It's actually going to be O(1) in this case because we are inserting at a specific index, which is the last index in this case